internal/oci/container.go

```diff
@@ -35,6 +35,9 @@ var (
 	ErrContainerStopped = errors.New("container is already stopped")
 	ErrNotFound         = errors.New("container process not found")
 	ErrNotInitialized   = errors.New("container PID not initialized")
+	ErrNotRunning       = errors.New("container not running")
+	ErrNoSuchProcess    = errors.New("no such process")
+	ErrInvalidProcess   = errors.New("invalid process")
```

Comment:
Since we already have `ErrNotFound` and use it, we can keep it and reuse it for our needs, too.

See some other comments.